apps/docs/content/components/button.mdx

```diff
@@ -4,4 +4,9 @@ description: The button Component
 status: "Released"
 ---
 
-# Button
+## Usage
+
+```jsx
```

Comment:
rehype-pretty-code is not running here. Something to look into as currently other mdx files are parsed and 
gets an better experience, syntax highlighting as well as a copyCode button.

 <img width="1022" alt="image" src="https://github.com/gsoft-inc/wl-hopper/assets/361632/d9d47ce2-50b0-433d-97ec-9d458de8eac3">
<img width="811" alt="image" src="https://github.com/gsoft-inc/wl-hopper/assets/361632/9ea4cff6-5d10-4382-a283-93b4aa600e9a">

Reply:
Done